### DISCUSSION TITLE
Quick Filter customize

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5662] [Help]
![image](https://user-images.githubusercontent.com/24252615/116110710-9b093100-a6be-11eb-8822-b039136e99ce.png)
I want to filter the stock name
I want to bring the ones with category id of stock. How can I do or is there any sample?
![image](https://user-images.githubusercontent.com/24252615/116111179-110d9800-a6bf-11eb-8d89-f1fc36ded7eb.png)

**Comment by edwardch** [comment]
Take a look at [previous issues for "filtervalue" or "filterfield"](https://github.com/serenity-is/Serenity/search?q=filtervalue%2C+filterfield&type=issues). It's similar to cascadeField and cascadeValue...
